src/i18n.js

```diff
@@ -75,14 +75,17 @@ const I18NManager = ({languageSelectorTarget, children}) => {
 
   const {messages, formioTranslations} = value;
 
-  const onLanguageChangeDone = newLanguageCode => {
+  const logLanguageChangeDone = newLanguageCode => {
+    onLanguageChangeDone(newLanguageCode);
     console.log(`Changed language to: ${newLanguageCode}`);
   };
```

Comment:
we can drop the `console.log`, so you can just forward the `onLanguageChangeDone` prop